README.md

```diff
@@ -2,105 +2,145 @@
 
 ![Stride Editor](https://stride3d.net/images/external/script-editor.png)
 
-To learn more about Stride, visit [stride3d.net](https://stride3d.net/).
+## Table of Contents
 
-## Earn money by contributing
-If you are a developer with solid experience in C#, rendering techniques, or game development, we want to hire you! We have allocated funds from supporters on [OpenCollective](https://opencollective.com/stride3d) and can pay for work on certain projects. [More info about this here](https://doc.stride3d.net/latest/en/contributors/engine/bug-bounties.html).
+* 🚀 [Getting Started](#-getting-started)
+* 🤝 [Contributing](#-contributing)
+  - [Earn Money by Contributing](#earn-money-by-contributing)
+* 🗺️ [Roadmap](#%EF%B8%8F-roadmap)
+* 🛠️ [Building from Source](#%EF%B8%8F-building-from-source)
+  - [Prerequisites](#prerequisites)
+  - [Build Stride](#build-stride)
+  - [Build Stride without Visual Studio](#build-stride-without-visual-studio)
+  - [If Building Fails](#if-building-fails)
+  - [Contribution Guidelines](#contribution-guidelines)
+* 🔬 [Build Status](#-build-status)
+* 📖 [Stride Documentation Landscape](#-stride-documentation-landscape)
+* 🌐 [.NET Foundation](#-net-foundation)
+* 🛡️ [License](#%EF%B8%8Flicense)
+* ✨ [Contributors](#-contributors)
 
-## License and governance
-### .NET Foundation
-This project is supported by the [.NET Foundation](https://dotnetfoundation.org).
+## 🚀 Getting Started
 
-### License
-Stride is covered by the [MIT License](LICENSE.md) unless stated otherwise (i.e. for some files that are copied from other projects). You can find the list of third-party projects [here](THIRD%20PARTY.md). Contributors need to sign the following [Contribution License Agreement](https://github.com/dotnet-foundation/.github/blob/main/CLA/dotnetfoundation.yml).
+If you want to **create games using Stride**, please visit our [Get started with Stride](https://doc.stride3d.net/latest/en/manual/get-started/index.html) guide. This guide provides detailed instructions on how to download, install, and begin using Stride to develop your games.
+
+If you are interested in **building the Stride engine from source** or **contributing to its development**, please continue reading this README for instructions on how to build from source and contribute to the project.
+
+If you are interested in both, we recommend starting with the [Get started with Stride](https://doc.stride3d.net/latest/en/manual/get-started/index.html) guide, and then returning here to learn how to build from source and contribute.
```

Comment:
This section is *really* long for how redundant it is, people will very likely just scan through the different headings we have to find what they are looking for instead of reading this.

Reply:
I think, this one is important for new comers. We can try to make it shorter and clearer. The aim is to figure out quickly the path one should take, as our current README is not clear and it was also confusing for myself. 

I didn't know if I need to follow README if I want to make just a game, and if I need to build from the source to make a game. So there are at least 2 major audiences with different goals.

So the **Getting Started** section should help with what one want to achieve:

1. Just creating games (not interested in Stride source code) - follow this path
2. Building, debugging, improving and contributing the Stride source code - continue reading

I will try to redraft that section. Any suggestions anyone?

Reply:
That makes sense, perhaps building a short warning/info blurb mentioning that the content of the readme is specifically for users wanting to build or contribute to the engine itself, and redirect users that want to create games to the documentation instead.

Reply:
Done.